librz/asm/asm.c

```diff
@@ -87,6 +87,14 @@ static bool is_register(const char *name, RZ_BORROW const RzRegSet *regset) {
 	return false;
 }
 
+static bool is_not_unknown(const char *str, size_t prev, size_t next) {
+	rz_return_val_if_fail(str, false);
+	// If the prev byte is not an operator or a separator and next byte is NULL(eg: "push rsp") ,don't consider it as unknown
+	// If the prev byte or next byte is not an operator or a separator, don't consider it as unknown.
+	return (is_operator(str + prev - 1) || is_separator(str + prev - 1)) &&
```

Comment:
Please document this function. The comment you already wrote here can be move to brief + descriptions of the parameters.